CMakeLists.txt

```diff
@@ -151,6 +189,13 @@ set(Launcher_BUILD_TIMESTAMP "${TODAY}")
 
 ################################ 3rd Party Libs ################################
 
+if(NOT Launcher_FORCE_BUNDLED_LIBS)
+    find_package(ZLIB QUIET)
+endif()
+if(NOT ZLIB_FOUND)
+    set(FORCE_BUNDLED_ZLIB 1)
+endif()
```

Comment:
Can probably be dropped

Reply:
I was mainly following the example set here; https://github.com/PrismLauncher/PrismLauncher/blob/8d39a9ea6185eacb79dc30a36a9d8e8fcc631528/CMakeLists.txt#L163-L166 but yeah It did strike me as redundant, I'll remove that for zlib shortly.